packages/eslint-plugin/src/rules/promise-function-async.ts

```diff
@@ -1,7 +1,9 @@
 import {
   AST_NODE_TYPES,
+  AST_TOKEN_TYPES,
   TSESTree,
 } from '@typescript-eslint/experimental-utils';
+import { last } from 'lodash';
```

Comment:
we'd rather avoid adding a dependency on lodash for such trivial things.
Please inline the function impl!